.gitignore

```
@@ -69,3 +69,6 @@ lock.mdb
 
 # Homebrew bundle
 /Brewfile.lock.json
+
+# pycache
+**/__pycache__
```

This doesn't appear to be working, `tools/fog-local-network/__pycache__/local_fog.cpython-36.pyc` is part of the PR.

```suggestion
__pycache__
```

---

Saw that the __pycache__ exclusion is already listed earlier in the .gitignore